src/HasteModuleLoader/HasteModuleLoader.js

```diff
@@ -113,6 +112,9 @@ function _getCacheFilePath(config) {
 
 function Loader(config, environment, resourceMap) {
   this._config = config;
+  this._CoverageCollector = require(
+    config.testCollector || '../CoverageCollector'
```

Comment:
Default config values can be set here:
https://github.com/facebook/jest/blob/master/src/lib/utils.js#L15